src/content/doc-surrealql/functions/database/type.mdx

```diff
@@ -1002,15 +1006,26 @@ RETURN type::is::polygon("I am not a polygon");
 
 false
 ```
+## `type::is::range`
 
-<br />
+The `type::is::range` function checks if the passed value is of type `range`.
+
+```surql title="API DEFINITION"
+type::is::range(any) -> bool
+```
+
+```surql
+(0..1).is_range();
+
+true
```

Comment:
```suggestion
```surql
type::is::range(0..1);
true
// method syntax
(0..1).is_range();

true
```